coalib/output/ConsoleInteraction.py

```diff
@@ -250,7 +252,9 @@ def print_result(console_printer,
     console_printer.print(format_lines(result.message))
 
     if interactive:
-        cli_actions = CLI_ACTIONS
+        # FIXME: Handle duplicate actions of same type across
+        # NATIVE_ACTIONS and result.actions
+        cli_actions = NATIVE_ACTIONS + tuple(result.actions)
```

Comment:
Also, I'm confused regarding handling these `NATIVE_ACTIONS` for this PR's use case.
1. Should we allow users to pass native actions like `OpenEditorAction` in the `actions` field.
2. If yes, then how do we handle multiple occurrences of such actions in `result.actions`
3. Should the users be allowed to define `next_actions` for these Native actions, and if yes, how are they going to do so.

This issue becomes complex when we combine 1,2 and 3 points. So I think we'll need to design it properly.